### PR TITLE
change providerUrl -> providerUrls

### LINKED ISSUE #144: Fix to work with latest Timenode-core develop
- Change `providerUrl` to take an array of string, `providerUrls`

### PR DESCRIPTION
**Pull request by lsaether**
fixes #144 
Depends on the new release of `timenode-core`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
   },
   "homepage": "https://github.com/ethereum-alarm-clock/eac.js-cli#readme",
   "dependencies": {
-    "@ethereum-alarm-clock/timenode-core": "5.2.1",
+    "@ethereum-alarm-clock/timenode-core": "^5.2.2",
```
Comment (Bagaric) [inline package.json]: Remove non-exact versioning.

File: src/program.js [hunk 2]
```diff
@@ -86,6 +86,7 @@ program
   .option('--minProfitability <eth>', 'Only claim transactions with a bounty higher', Config.DEFAULT_ECONOMIC_STRATEGY.minProfitability.div(Math.pow(10, 18)))
   .option('--maxGasSubsidy <eth>', 'Subsidize a percentage of gas costs on gas spikes', Config.DEFAULT_ECONOMIC_STRATEGY.maxGasSubsidy)
   .option('--ms <number>', 'Sets the scanning frequency of the TimeNode', 4000)
+  .option('--providers <list_of_providers>', '')
```
Comment (Bagaric) [inline src/program.js]: How does one provide this array? A string separate by something?

Comment (lsaether) [reply src/program.js]: Oh whoops. Yeah it should be a string of the addresses separated by commas, no spaces.

Comment (Bagaric) [reply src/program.js]: Please add a description too. Keep in mind that commas can be a part of an URL. We might want to use a different separator.

Comment (lsaether) [reply src/program.js]: Hm... That's true, although they're rarely used. Adding in Config compatibility too in the case where the comma separated string doesn't work.

File: sampleConfig.json [hunk 3]
```diff
@@ -8,6 +8,9 @@
     "maxGasSubsidy": 100,
     "password": "password",
     "provider": "http://localhost:8545",
+    "providers": [
+
```
Comment (Bagaric) [inline sampleConfig.json]: Unnecessary empty line.

Comment (lsaether) [reply sampleConfig.json]: fixed in latest commit

**Pull request merged**